src/unix-manager.c

```diff
@@ -414,9 +426,16 @@ static int UnixCommandAccept(UnixCommand *this)
 
     json_decref(server_msg);
 
+    if (client_version > UNIX_PROTO_V1) {
+        if (send(client, "\n", 1, MSG_NOSIGNAL) == -1) {
```

Comment:
Is it possible to just oversize the buffer on UNIX_PROTO_V2 or higher, and after write set the write length + 1 to the return, rather than sending the return as an additional send?

Reply:
Yes, make sense indeed, as pointed by @inliniac this code is quite old and was done before the code was switched to use a memory buffer. Gonna fix that.